src/main/java/org/mifos/connector/mockPaymentSchema/schema/BatchDTO.java

```diff
@@ -0,0 +1,226 @@
+package org.mifos.connector.mockPaymentSchema.schema;
+
+import java.math.BigDecimal;
+
+public class BatchDTO {
+
+    private String batch_id;
+
+    private String request_id;
+
+    private Long total;
+
+    private Long ongoing;
+
+    private Long failed;
+
+    private Long successful;
+
+    private BigDecimal totalAmount;
+
+    private BigDecimal successfulAmount;
+
+    private BigDecimal pendingAmount;
+
+    private BigDecimal failedAmount;
+
+    private String file;
+
+    private String notes;
+
+    private String created_at;
+
+    private String status;
+
+    private String modes;
+
+    private String purpose;
+
+    private String failPercentage;
+
+    private String successPercentage;
+
+    public String getFailPercentage() {
+        return failPercentage;
+    }
+
+    public void setFailPercentage(String failPercentage) {
+        this.failPercentage = failPercentage;
+    }
+
+    public String getSuccessPercentage() {
+        return successPercentage;
+    }
+
+    public void setSuccessPercentage(String successPercentage) {
+        this.successPercentage = successPercentage;
+    }
+
+    public BatchDTO(){}
+
+    public BatchDTO(String batchId, String requestId, Long totalTransactions, Long ongoing, Long failed, Long completed, BigDecimal total_amount, BigDecimal completed_amount, BigDecimal ongoing_amount, BigDecimal failed_amount, String result_file, String note, String failPercentage, String successPercentage) {
+        this.batch_id = batchId;
+        this.request_id = requestId;
+        this.total = totalTransactions;
+        this.ongoing = ongoing;
+        this.failed = failed;
+        this.successful = completed;
+        this.totalAmount = total_amount;
+        this.successfulAmount = completed_amount;
+        this.pendingAmount = ongoing_amount;
+        this.failedAmount = failed_amount;
+        this.file = result_file;
+        this.notes = note;
+        this.failPercentage = failPercentage;
+        this.successPercentage = successPercentage;
+    }
+
+    public BatchDTO(String batch_id, String request_id, Long total, Long ongoing, Long failed, Long successful, BigDecimal totalAmount, BigDecimal successfulAmount, BigDecimal pendingAmount, BigDecimal failedAmount, String file, String notes, String created_at, String status, String modes, String purpose, String failPercentage, String successPercentage) {
```

Comment:
instead of adding two conductors, create two factory methods with a proper name.
here its difficult to understand the use case of these constructors